packages/react-components/src/lib/cesium-map/data-sources/drawings.data-source.stories.tsx

```diff
@@ -84,8 +86,62 @@ export const Drawings: Story = (args) => {
       >
         Box
       </button>
+      <button
+        style={{ position: 'fixed', top: '80px', left: '20px', zIndex: 1 }}
+        onClick={(): void => {
+          setIsDrawing(false);
+          setDrawPrimitive({
+            type: DrawType.UNKNOWN,
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            handler: (drawing: IDrawingEvent) => {},
+          });
+        }}
+      >
+        Stop Draw
+      </button>
+      <button
+        style={{ position: 'fixed', top: '140px', left: '20px', zIndex: 1 }}
+        onClick={(): void => {
+          setIsDrawing(false);
+          setDrawEntities([
+            {
+              coordinates: undefined,
```

Comment:
why isn't it nullable same as with the geojson attribute?

Reply:
This data-strucutre mainly used as output event from "drawing primitive" and **coordinates** is compulsory.
 Want to emphasize in case of "undefined" is a same data structure in the way IN to add primitive to layer instead of drawing it. And coordinates in this case should be explicitly undefined rather than **optional**

https://stackoverflow.com/questions/47942141/optional-property-class-in-typescript